app/version.py

```diff
@@ -1,2 +1,2 @@
-__git_commit__ = "61b9613c6b1735ad24a3c14eca4830bd44575da4"
-__time__ = "2022-03-24:09:01:25"
+__git_commit__ = "0477d9433726e05c9d88120a5112f90e2a96afcf"
+__time__ = "2022-03-30:14:50:08"
```

Comment:
What did you do to change this?

Reply:
I have to double check but I think this was part of the local git flow. I didn't manually change this myself.

Reply:
I reverted this file back to what was in `master`. If this file does not need to be I the repo then we need to have it back in the `.gitignore` file and note in the `README.md` that it is required if it isn't already generated in some way.